src/tests/ftest/launch.py

```diff
@@ -52,2294 +68,2973 @@
 
-    Returns:
-        int: status code: 0 = success, 512 = failure
+        Args:
+            hosts (NodSet): hosts on which the files are located
+            source (str): where the files are currently located
+            pattern (str): pattern used to limit which files are processed
+            depth (int): max depth for find command
 
-    """
-    service = "daos_server.service"
-    hosts = get_hosts_from_yaml(test_file["yaml"], args, YAML_KEYS["test_servers"])
-    log("-" * 80)
-    log("Verifying {} on {} after running '{}'".format(service, hosts, test_file["py"]))
-    return stop_service(hosts, service)
+        Returns:
+            int: status code: 0 = success, 16 = failure
 
+        """
+        logger.debug("-" * 80)
+        logger.debug(
+            "Compressing any %s files in %s on %s larger than 1M", pattern, source, hosts)
+        other = ["-size", "+1M", "-print0", "|", "sudo", "-n", "xargs", "-0", "-r0", "lbzip2", "-v"]
+        result = run_remote(logger, hosts, find_command(source, pattern, depth, other))
+        if not result.passed:
+            message = f"Error compressing {os.path.join(source, pattern)} files larger than 1M"
+            self._fail_test(self.result.tests[-1], "Process", message)
+            return 16
+        return 0
+
+    def _move_files(self, hosts, source, pattern, destination, depth, timeout):
+        """Move files from the source to the destination.
 
-def stop_service(hosts, service):
-    """Stop any daos_server.service running on the hosts running servers.
+        Args:
+            hosts (NodSet): hosts on which the files are located
+            source (str): where the files are currently located
+            pattern (str): pattern used to limit which files are processed
+            destination (str): where the files should be moved to on this host
+            depth (int): max depth for find command
+            timeout (int): number of seconds to wait for the command to complete.
 
-    Args:
-        hosts (NodeSet): list of hosts on which to stop the service.
-        service (str): name of the service
+        Returns:
+            int: status code: 0 = success, 16 = failure
 
-    Returns:
-        int: status code: 0 = success, 512 = failure
+        """
+        logger.debug("-" * 80)
+        logger.debug("Moving files from %s to %s on %s", source, destination, hosts)
+
+        # Core and dump files require a file ownership change before they can be copied
+        if "stacktrace" in destination or "daos_dumps" in destination:
+            # pylint: disable=import-outside-toplevel
+            other = ["-print0", "|", "xargs", "-0", "-r0", "sudo", "-n", get_chown_command()]
+            if not run_remote(logger, hosts, find_command(source, pattern, depth, other)).passed:
+                message = f"Error changing {os.path.join(source, pattern)} file permissions"
+                self._fail_test(self.result.tests[-1], "Process", message)
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'fail_reason' in staticmethod call